### DISCUSSION TITLE
quarkus-resteasy-reactive-kotlin as optional dependency

### DISCUSSION THREAD

**Discussion by paloliska** [quarkusio/quarkus#35088] [Q&A] [answered]
Hello, 
I spotted that extension `quarkus-resteasy-reactive-kotlin` is defined as `<optional>true</optional> <!-- conditional dependency -->`.

Does is mean that I need to specify it explicitly in my pom?

And what about `quarkus-resteasy-reactive-kotlin-serialization`?

[a comment by an automated account is not shown]

**Comment by geoand** [comment]
Where exactly did you see that?

**Reply by paloliska** [reply]
https://github.com/quarkusio/quarkus/blob/45ed1c7cd94b6c737823d7e1813ac382a8b7fa9a/extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/pom.xml#L23

**Comment by geoand** [comment, marked as answer]
Quarkus will automatically add `quarkus-resteasy-reactive-kotlin` if `quarkus-kotlin` and `quarkus-resteasy-reactive` are present.

`quarkus-resteasy-reactive-kotlin-serialization` brings in both and adds Kotlin serialization support

**Reply by paloliska** [reply]
ok thanks
